staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4267,6 +4267,32 @@ type PodResourceClaimStatus struct {
 	ResourceClaimName *string `json:"resourceClaimName,omitempty" protobuf:"bytes,2,opt,name=resourceClaimName"`
 }
 
+// PodExtendedResourceClaimStatus is stored in the PodStatus for the extended
+// resource requests backed by DRA. It stores the generated name for
+// the corresponding special ResourceClaim created by the scheduler.
+type PodExtendedResourceClaimStatus struct {
+	// RequestMapping identifies the mapping of <container, extended resource backed by DRA> to  device request
+	// in the generated ResourceClaim.
+	// +listType=atomic
+	// +listMapKey=requestName
+	RequestMapping []ContainerExtendedResourceRequest `json:"requestMapping" protobuf:"bytes,1,rep,name=requestMapping"`
```

Comment:
If each item in the list is a request mapping, should this be `requestMappings`?

Reply:
Yes, renamed to RequestMappings